webapp/src/plugin.jsx

```diff
@@ -52,16 +52,20 @@ export default class DemoPlugin {
         registry.registerBottomTeamSidebarComponent(
             BottomTeamSidebar,
         );
-        const {showRHSPlugin} = registry.registerRightHandSidebarComponent(
+        const {showRHSPlugin, hideRHSPlugin} = registry.registerRightHandSidebarComponent(
             RHSView,
             <FormattedMessage
                 id='plugin.name'
                 defaultMessage='Demo Plugin'
             />);
 
+        let actionButtonIsActive = false;
```

Comment:
My ideal API is to have the equivalent of `showMe`, `hideMe`, and `toggleMe`. `showMe` would replace any existing RHS and open my own registered component. `hideMe` would hide the RHS only if I was active. `toggleMe` would be a `showMe` if already shown and `hideMe` otherwise.

Would it be feasible to update https://github.com/mattermost/mattermost-webapp to expose three methods as such?

Reply:
Yeah, sounds reasonable. I can do it, I can just update this MR. right?
And I'll replace this condition with just toggle action, is it ok to have only one of three in example plugin?

Reply:
Yes, you can push more changes to this PR and the webapp as needed. 

It should be fine to only use one of the three actions for the RHS in the demo plugin. Thanks @Lisenish!

Reply:
@lieut-data @marianunez Guys, sorry for the delay I had a very busy week. I've made changes in both repos, now button is working nice even if you open another panel. Please take a look.

Reply:
No worries @Lisenish! Thanks for the changes 🎉